daemon/index_document.h

```diff
@@ -42,50 +42,73 @@ class index_document : public index_info
         virtual ~index_document() throw ();
```

Comment:
Just wondering? Is this code ever executed? I assumed there is no indexing for documents...